modelator/src/cli/mod.rs

```diff
@@ -74,55 +75,74 @@ pub struct TraceCli {
     /// TLA+ config file with CONSTANTS, INIT and NEXT.
     #[clap(parse(from_os_str), value_hint = ValueHint::FilePath)]
     tla_config: PathBuf,
+    /// The number of traces to generate for a single test.
+    #[clap(long)]
```

Comment:
add this attribute `#[clap(short, long, default_value = "0")]` and change the type to `usize` (number of trace is positive integer). also move this field before `tla_module` field.

Reply:
Good points, thanks.